src/managers/base.ts

```diff
@@ -8,7 +8,7 @@ import { Collection } from '../utils/collection.ts'
  */
 export class BaseManager<T, T2> {
   client: Client
-  /** Cache Name or Key used to differentiate caches */
+  /** Caches Name or Key used to differentiate caches */
```

Comment:
I think it was right there before.. lol or you mean "cache's"?

Reply:
I thought the word cache is used as a verb at the front, so I changed the from